tools/depends/target/python36/Python-3.6.2-android.patch

```diff
@@ -40,3 +40,16 @@
      {"tcflush", termios_tcflush,
       METH_VARARGS, termios_tcflush__doc__},
      {"tcflow", termios_tcflow,
+
+     
```

Comment:
I thought it would make it easier to differentiate between files in the same patch file.